Source/SwiftyDropbox/Shared/Handwritten/OAuth/OAuth.swift

```diff
@@ -642,7 +642,7 @@ class Keychain {
 
         if status == noErr {
             let results = dataResult as? [[String : AnyObject]] ?? []
-            return results.map { d in d["acct"] as! String }
+            return results.map { d in (d["acct"] as? String) ?? "" }
```

Comment:
@YufeiG @xuyanke without this change (which is the only changed needed to the SDK), the macOS tests crash here. The result contains many more elements on iOS then on macOS, and one of those elements is "acct".